armi/nucDirectory/nuclideBases.py

```diff
@@ -825,7 +445,7 @@ def getDecay(self, decayType):
         return None
 
     def isFissile(self):
-        r"""Determine if the nuclide is fissile.
+        """Determine if the nuclide is fissile.
 
         Determines if the nuclide is fissle.
```

Comment:
This part of the docstring is duplicated.

Reply:
Fixed.